solidity/random-beacon/README.adoc

```diff
@@ -93,10 +92,10 @@ performed.
 The sortition pool weights operators by their authorized stake amount and allows
 selecting the same operator multiple times. Off-chain DKG protocol executes in
 the same way as for v1 and inactive/disqualified members during the off-chain
-protocol are marked as ineligible for rewards for a certain, governable, period
+protocol are marked as ineligible for rewards for a certain period
```

Comment:
Is this period controlled by `sortitionPoolRewardsBanDuration` variable? If so, could we add add this case to the description of the parameter in the Parameters table?